data/json/faults.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
```

Comment:
This is the same format I've followed for everything else and I'm a long way through much of the JSON now (see under `items/`). I'm not sure indentation is that important providing it's consistent within a file. Alphabetical ordering of entries by `id` is the more major factor.